project/Build.scala

```diff
@@ -127,13 +127,19 @@ object ScalajsReactComponents extends Build {
     .dependsOn(macros)
     .settings(
       name := "core",
+      addCompilerPlugin(compilerPlugin("org.scalamacros" % "paradise" % "2.0.1" cross CrossVersion.full)),
       libraryDependencies ++= Seq(
         "com.github.japgolly.scalajs-react" %%% "core" % scalajsReactVersion,
         "com.github.japgolly.scalajs-react" %%% "extra" % scalajsReactVersion,
+        "com.github.japgolly.scalajs-react" %%% "ext-monocle" % scalajsReactVersion,
```

Comment:
hmm, first of all why we need them here ? , core shouldn't depend on scalaz/monocle  ,event scalajs-react core/extra doesn't depend on scalaz etc